Add files via upload

Artem22831 opened the pull request.

Artem22831 committed:
Add files via upload

Artem22831 committed:
Add files via upload

Artem22831 committed:
Add files via upload

Nastasia8 commented on 1 zadanie sortirovka.txt:
```diff
@@ -0,0 +1,18 @@
+s = input()
+a = []
+k = 0
+s = s.split(" ")
```
Только лучше сразу s = input().split(" ") -> код на 2 строке лишний

Nastasia8 replied:
А так, ок

Nastasia8 commented on 5 zadanie kol.razlichnih.txt:
```diff
@@ -0,0 +1,4 @@
+n = int(input())
+a = map(int, input().split())
+a = set(a)
+print(len(a))
```
1, 5 - Ок

Nastasia8 commented on 4 zadanie inversia.txt:
```diff
@@ -0,0 +1,6 @@
+min=j.index(min(j))
+max=j.index(max(j[min:]))+min
+if min < max and max(j) / min(j) > 1: print (min+1, maxi-mini+1)
+else: print ('0')
```
Вы тестировали код?
На яндекс-контесте задачу приняли?)

Artem22831 commented:
Номер 4 в компиляторе работал ,а вот в Яндекс контекст почему то нет )

чт, 4 мар. 2021 г., 13:37 Nastasia8 <notifications@github.com>:

> *@Nastasia8* commented on this pull request.
> ------------------------------
>
> In 1 zadanie sortirovka.txt
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#discussion_r587342264>
> :
>
> > @@ -0,0 +1,18 @@
>
> +N = int(input())
>
> +s = input()
>
> +a = []
>
> +k = 0
>
> +s = s.split(" ")
>
>
> Только лучше сразу s = input().split(" ") -> код на 2 строке лишний
> ------------------------------
>
> In 5 zadanie kol.razlichnih.txt
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#discussion_r587351249>
> :
>
> > @@ -0,0 +1,4 @@
>
> +n = int(input())
>
> +a = map(int, input().split())
>
> +a = set(a)
>
> +print(len(a))
>
>
> 1, 5 - Ок
> ------------------------------
>
> In 4 zadanie inversia.txt
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#discussion_r587352487>
> :
>
> > @@ -0,0 +1,6 @@
>
> +i=input()
>
> +j=list(map(int, input().split()))
>
> +min=j.index(min(j))
>
> +max=j.index(max(j[min:]))+min
>
> +if min < max and max(j) / min(j) > 1: print (min+1, maxi-mini+1)
>
> +else: print ('0')
>
>
> Вы тестировали код?
> На яндекс-контесте задачу приняли?)
> ------------------------------
>
> In 1 zadanie sortirovka.txt
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#discussion_r587342340>
> :
>
> > @@ -0,0 +1,18 @@
>
> +N = int(input())
>
> +s = input()
>
> +a = []
>
> +k = 0
>
> +s = s.split(" ")
>
>
> А так, ок
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#pullrequestreview-603936098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARAEUTPFOLOA32PEKNHQG4LTB5PGFANCNFSM4YS3LGPQ>
> .
>

Nastasia8 commented:
А можете мне скриншот работы (4 программы) скинуть?
Для примера: кол-во элементов 5 (i = 5), а сами элементы 5 4 3 2 1

Artem22831 commented:
вот 4 номер

чт, 4 мар. 2021 г. в 14:31, Nastasia8 <notifications@github.com>:

> А можете мне скриншот работы (4 программы) скинуть?
> Для примера: кол-во элементов 5 (i = 5), а сами элементы 5 4 3 2 1
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#issuecomment-790547200>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARAEUTJPBRGNRLNZOYM4FBTTB5VPVANCNFSM4YS3LGPQ>
> .
>

Artem22831 committed:
Add files via upload

Artem22831 commented:
7 номер сделал,но там ошибка где-то,6 номер вообще незнаю как делать

вт, 9 мар. 2021 г. в 14:36, Jumper csgo <artembakiev77@gmail.com>:

> вот 4 номер
>
> чт, 4 мар. 2021 г. в 14:31, Nastasia8 <notifications@github.com>:
>
>> А можете мне скриншот работы (4 программы) скинуть?
>> Для примера: кол-во элементов 5 (i = 5), а сами элементы 5 4 3 2 1
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/27#issuecomment-790547200>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ARAEUTJPBRGNRLNZOYM4FBTTB5VPVANCNFSM4YS3LGPQ>
>> .
>>
>

Nastasia8 commented on 7 zadanie sortirovka.txt:
```diff
@@ -0,0 +1,3 @@
+B = [9, 12, 32, 45, 67, 98, 29, 61, 35, 09]
+sorted(B, key=int, reverse=True)
+print(*sorted(B))
```
Что это за магия?)

Nastasia8 replied:
На занятии посмотрим.
